typings/connection.d.ts

```diff
@@ -299,9 +282,14 @@ export interface TargetTerminusOptions extends BaseTerminusOptions {
  */
 export interface Source extends TerminusOptions {
   /**
-  * @property {number} [distribution_mode] The distribution mode of the link.
+  * @property {string} [distribution_mode] The distribution mode of the link.
+  * Valid values are:
+  * - **move** - once successfully transferred over the link, the message will no longer be
+  * available to other links from the same node.
+  * - **copy** - once successfully transferred over the link, the message is still available
+  * for other links from the same node.
   */
-  distribution_mode?: DistributionMode;
+  distribution_mode?: "move" | "copy";
```

Comment:
@GaikwadPratik - This will still provide validation in TS. The documentation should provide good intellisense in your IDE. I believe this is a better fix over enum. What do you think?